taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/parselistener/TaskanaParseListenerProcessEnginePlugin.java

```diff
@@ -19,9 +19,8 @@
 import pro.taskana.adapter.camunda.util.ReadPropertiesHelper;
 
 /**
- * Camunda engine plugin for the taskana parse listener.
- *
- * @author jhe
+ * Camunda engine plugin for the responsible for adding the TaskanaParseListener to the
```

Comment:
should read "Camunda engine plugin responsible for adding the TaskanaParseListener ..."